### PR TITLE
Switch to AGPL-3.0

### LINKED ISSUE #490: IMPORTANT LICENSE CHANGE
Hey @Starystars67 @20dka @snepsnepsnep @jojos38 @OfficialLambdax @stefan750 @Olrosse @Please-Pick-a-Name @vulcan-dev @StanleyDudek @scisdev @O1LER @Anonymous-275 @Mack29446 @prestonelam2003 @50thomatoes50 @2PXL @AlexITA1100 @0R3Z @Desoroxxx @YumiNee213 @JamesBlad @iTeeLion @ArmenG888 @1eob @jacobcargen @5TXdYWK @daveywaveyHD @VanilleVaschnille @rtkpX @NewZulund @Neptnium @x0xDJSx0x @NicholasMcMillan03,

We're changing the license of this codebase to AGPL-3.0.

Please respond with a short "Okay" or similar message of acknowledgement. If you have any issues or concerns, please make them known.

We will move ahead with the AGPL-3.0 change, we need to know who's code we need to remove for this.

### PR DESCRIPTION
**Pull request by lionkor**
See #490

- [x] Add license
- [x] Add license header to all files (if wanted)
- [x] Explicitly remove/state files which are not AGPL-3.0 (BeamNG files, if any)
- [x] Update NOTICE
- [x] Include 3rd party (BeamNG own and other) licenses
- [x] Review the passenger.lua again to see how much is from the game and how much is not.
- > The code is almost the same
- [x] Removal of `ui\modules\multiplayer\flags.png`
- > Unused and of unknown origin. To be done by @Starystars67
- [x] Removal of `ui/modules/multiplayer/css/timeline.css`
- > Unused and of unknown origin. To be done by @Starystars67
- [x] Final review and potential polish
- [x] Wait for the last 2 contributors to agree
- > https://github.com/BeamMP/BeamMP/issues/490

### CODE REVIEW

**Comment by lionkor** [comment]
@Starystars67 do we want to do what the license states at the bottom in terms of adding minimal headers to files?

**Comment by Starystars67** [comment]
> @Starystars67 do we want to do what the license states at the bottom in terms of adding minimal headers to files?

Yes, The same should also be done for any original game files if they are missing and changes documented where possible.

**Comment by lionkor** [comment]
Please if someone could self-assign this and finish it, that'd be great. I'm swamped with server, launcher and protocol

**Comment by OfficialLambdax** [comment]
Will handle the rest tomorrow (*if i dont forget*).
I will
1. [x] remove all present file header notices and simply put a link to this repo while also stating its AGPL 3.0
2. [x] Make exceptions for all files that we patch from the game (mostly if not all just UI files)
3. [x] Add the game own files to the notices
4. [x] Do ui headers

**Comment by lionkor** [comment]
File headers should really be the one that I put on the other files

**Comment by OfficialLambdax** [comment]
Note. adding comment headers to .html files of angularjs apps can break the loading of these

**Comment by Starystars67** [comment]
@lionkor @OfficialLambdax are you both ready with this now?

**Comment by lionkor** [comment]
I would like to review this tonight and then I'll merge

File: NOTICES.md [hunk 1]
```diff
@@ -16,12 +16,24 @@ All code, assets or works not specified in this document which do not have a dif
...
-  - **License:** bCDDL
-  - **Location:** https://beamng.com/bCDDL-1.1.txt
-  - **Files:** 
-    - ``
+- GameAssets: By BeamNG GmbH
+  - **License:** Common Development and Distribution License
+  - **Location 1:** `./BeamNG.drive/lua/bCDDL-1.1.txt`
+  - **Location 2:** https://beamng.com/bCDDL-1.1.txt
+  - **Files:**
+    - `lua/ge/extensions/core/cameraModes/passenger.lua`
```
Comment (snepsnepsnep) [inline NOTICES.md:25]: passenger.lua is not a direct game asset, it's a custom camera mode made for beammp derived from driver.lua (which is a game asset).

Comment (OfficialLambdax) [reply NOTICES.md:25]: correct and as such the code found in passenger.lua underlays the bCDDL license and should be noted there, no?

**Comment by OfficialLambdax** [comment]
If there are no objections within the next 24hrs i will rereview this pr to make sure that everything is set and correct and then merge

**Comment by lionkor** [comment]
Scisdev's code isn't in the repo anymore as far as we could tell, and 20dka agreed

**Comment by lionkor** [comment]
but maybe @20dka just write it in the linked issue for completeness' sake

**Review by OfficialLambdax — APPROVED** [review]
All tasks have been completed. Each file has been look at twice. Respective 3rd party assets have been indexed within the notices.md and as far as possible also given their respective license header in the files themself. And everyone has agreed to make this change (:

**Pull request merged**